src/index.tsx

```diff
@@ -4,6 +4,9 @@ import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
 
+import { worker } from './mocks/server';
+worker.start();
```

Comment:
I don't think starting the mock server directly is a right choice. The mock's startup should be binded with situation flag like ENV=dev etc.